code/game/objects/items/weapons/RCD.dm

```diff
@@ -11,312 +11,180 @@ RCD
 	icon_state = "rcd"
 	opacity = 0
 	density = 0
-	anchored = 0
+	anchored = 0.0
```

Comment:
I'm not sure we need to say anchored is a decimal, unless it's used like this elsewhere